mne/io/ctf/ctf.py

```diff
@@ -165,6 +165,34 @@ def _read_segment_file(self, data, idx, fi, start, stop, cals, mult):
                 _mult_cal_one(data_view, this_data, idx, cals, mult)
                 offset += n_read
 
+    def _clean_names(self):
+        """Clean up CTF suffixes from channel names."""            
+        for comp in self.info['comps']:
+            for key in ('row_names', 'col_names'):
+                comp['data'][key] = _clean_names(comp['data'][key])
+
+    def rename_channels(self, mapping, clean_channel_names=True):
+        """Rename channels and clean up CTF data if needed.
+
+        Parameters
+        ----------
+        mapping : dict | callable
+            a dictionary mapping the old channel to a new channel name
+            e.g. {'EEG061' : 'EEG161'}. Can also be a callable function
+            that takes and returns a string (new in version 0.10.0).
+
+        clean_channel_names : bool, optional
+            a bool patameter, if it equals to True channel names cleaning
+            will be performed.
+
+        Notes
+        -----
+        .. versionadded:: 0.16.0
+        """
+        super(RawCTF, self).rename_channels(mapping)
```

Comment:
does it work if mapping is empty?

Reply:
No, it will not work. The ValueError exception will be raised by the (rename_channels function)[https://github.com/mne-tools/mne-python/blob/master/mne/channels/channels.py#L983]

Reply:
so i don't get the logic here. How am I supposed to clean up the CTF channel names using public API?

you should edit an example using CTF data to document this and see how an enduser will make use of this.

Reply:
In the original issue #4207 two possible ways were suggested to solve the problem: 
1. modify raw.rename_channels function, so it will be able to clean channel names;
2. add new raw.clean_names function that will be concerned only about cleaning CTF channels names.

As far as I understand, the use case, described in that issue, assumes that this new functionality will be used alongside super().rename_channels function.

Do you think second option will be more suitable? Or should I provide a fallback in case of empty   mappings to ctf_raw.rename_channels function, e. g. `mapping = dict(zip(raw.ch_names, mne.utils._clean_names(raw.ch_names)))`?

@dengemann what will better suit your needs like end-user?